app/assets/v2/js/pages/profile-trust.js

```diff
@@ -751,6 +873,166 @@ Vue.component('active-trust-row-template', {
   `
 });
 
+Vue.component('ens-verify-modal', {
+  delimiters: [ '[[', ']]' ],
+  data: function() {
+    return {
+      ethAddress: '',
+      static_url: document.contxt.STATIC_URL,
+      ensDomain: '',
+      validationStep: 0,
+      validationError: '',
+      validationErrorMsg: '',
+      url: '',
+      validated: false,
+      verificationEthAddress: ''
+    };
+  },
+  props: {
+    showValidation: {
+      type: Boolean,
+      required: false,
+      'default': false
```

Comment:
I believe is without quotes I never saw it with quotes, but probably doesn't affect the code.